scala-package/scalastyle-config.xml

```diff
@@ -64,7 +64,7 @@ You can also disable only one rule, by specifying its rule id, as specified in:
 
   <check level="error" class="org.scalastyle.file.FileLineLengthChecker" enabled="true">
     <parameters>
-      <parameter name="maxLineLength"><![CDATA[100]]></parameter>
+      <parameter name="maxLineLength"><![CDATA[132]]></parameter>
```

Comment:
Undo this. I know you picked it up from my change but Naveen wanted us to have this discussion on the dev list before changing it.